sdk/core/azure-core/azure/core/pipeline/policies/_universal.py

```diff
@@ -340,6 +340,7 @@ class HttpLoggingPolicy(SansIOHTTPPolicy):
     """
 
     DEFAULT_HEADERS_WHITELIST = set([
+        "x-ms-request-id",
```

Comment:
Do we want to add x-ms-date, x-ms-version too?

Reply:
x-ms-date is storage specific https://github.com/Azure/azure-sdk-for-python/issues/10960
We didn't discuss x-ms-version, but I would thinkg it's storage specific too

Reply:
From the request:

Describe the solution you'd like
A clear and concise description of what you want to happen.

Can we add x-ms-request-id, x-ms-date, x-ms-version in whitelist? Because it is very useful for users to debug and these information will be output when error occurs as shown here, in this way it should be fine to add these information in whitelist.

Is "x-ms-request-id" good enough?

Reply:
For azure-core yes, I forgot x-ms-version I will add it to storage issue, thanks!